pkgs/development/python-modules/mathlibtools/default.nix

```diff
@@ -11,7 +11,7 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [
-    PyGithub GitPython toml click tqdm paramiko networkx pydot pyyaml
+    PyGithub GitPython toml click tqdm paramiko networkx pydot pyyaml atomicwrites
```

Comment:
paramiko can be removed.

https://github.com/leanprover-community/mathlib-tools/commit/0eec4cc1cd7057e79f15b3815392352f15487ecc#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

Reply:
Good catch!